src/elm/clinical.ts

```diff
@@ -158,11 +168,13 @@ export class Code extends Expression {
   version: string;
   display?: string;
 
-  constructor(json: any) {
+  constructor(json: ELM.Code) {
     super(json);
     this.code = json.code;
-    this.systemName = json.system.name;
-    this.version = json.version;
+    this.systemName = json.system?.name || '';
+    // @ts-ignore
```

Comment:
add eslint comment

Reply:
Or maybe we should just tell eslint to allow `@ts-ignore`?  If we know we need to use it in a bunch of places...